src/main/resources/liquibase/user/changelog_1.0.xml

```diff
@@ -7,7 +7,7 @@
             CREATE SCHEMA IF NOT EXISTS users
         </sql>
     </changeSet>
-    <changeSet author="Dziyana Bahdanava" id="create table Users">
+    <changeSet author="Dziyana Bahdanava" id="create table users">
```

Comment:
just for your information, if you have executed your changeSet once then it's persisted in the database along with this id (create table Users). If you then change you id (create table **u**sers) then liquibase will consider it as a new script and try to apply it to your database one more time

Reply:
Yes, unfortunately, I had the opportunity to understand this😂